### PR TITLE
Student profile FE

### PR DESCRIPTION
**Pull request by nicklee**
Contains:

https://projects.torchbox.com/projects/rca-website-rebuild/tickets/654
https://projects.torchbox.com/projects/rca-website-rebuild/tickets/633

Based on the staff profile page that we already have

### CODE REVIEW

File: rca/project_styleguide/templates/patterns/molecules/job-titles/job-titles.html [hunk 1]
```diff
@@ -4,13 +4,15 @@
             {% for role in group.items %}
                 <li class="job-titles__title">{{ role }}</li>
             {% endfor %}
-            <li class="job-titles__department">
-            {% if group.link %}
-                <a class="body body--two link link--tertiary" href="{{ group.link }}">{{ group.label }}</a>
-            {% else %}
-                <p class="body body--two link">{{ group.label }}</p>
+            {% if group.link or ground.label %}
```
Comment (chris-lawton) [inline rca/project_styleguide/templates/patterns/molecules/job-titles/job-titles.html:7]: should this be `group.label`?

File: rca/project_styleguide/templates/patterns/pages/student/student_detail.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                {% if page.profile_image %}
+                    <div class="title-area__image">
+                        {% include "patterns/atoms/profile-image/profile-image.html" %}
+                    </div>
+                {% endif %}
+                <div class="title-area__content">
+                    <nav class="title-area__breadcrumb" aria-label="Breadcrumb">
+                        {% include "patterns/molecules/breadcrumb/breadcrumb.html" %}
+                    </nav>
+                    {% if page.staff_title or page.first_name or last_name %}
```
Comment (chris-lawton) [inline rca/project_styleguide/templates/patterns/pages/student/student_detail.html:27]: `or page.last_name`

Comment (nicklee) [reply rca/project_styleguide/templates/patterns/pages/student/student_detail.html:27]: good spot, shall fix in the staff template too!

File: rca/project_styleguide/templates/patterns/molecules/job-titles/job-titles.html (hunk 1 above)
Comment (nicklee) [reply rca/project_styleguide/templates/patterns/molecules/job-titles/job-titles.html:7]: oops, yes :) thanks 👍

**Pull request merged**